src/Core/Addon/Module/ModuleManager.php

```diff
@@ -423,11 +433,24 @@ public function disable_mobile($name)
      * Not written in camel case because the route and the displayed action in the template
      * are related to this function name.
      *
+     * @deprecated use enableMobile.
```

Comment:
This function was chosen. Each function name in this class is directly displayed or translated in the actions list, with underscores replaces by spaces.